multichain-testing/README.md

```diff
@@ -66,6 +67,9 @@ make port-forward
 
 # set up Agoric testing environment
 make fund-provision-pool override-chain-registry register-bank-assets
+
+# create and fund a liquidity pool on Osmosis
+make create-osmosis-pool
```

Comment:
Please move this to a new `# Optional Features` section

Reply:
Request addressed on commit [f25f551](https://github.com/Agoric/agoric-sdk/pull/11229/commits/f25f5517712a8d41792b45c29f34ede8acfc5804)